docs/getting-started-with-micro/index.md

```diff
@@ -0,0 +1,286 @@
+---
+title: "Setting up Snowplow Micro locally"
+sidebar_position: 25
+---
+
+```mdx-code-block
+import {versions} from '@site/src/componentVersions';
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Badges from '@site/src/components/Badges';
+```
+
+## What is Snowplow Micro
```

Comment:
Not entire sure where, but perhaps getting a nice, obvious, link to the API docs somewhere on this page would be good? https://deploy-preview-301--snowplow-docs.netlify.app/docs/pipeline-components-and-applications/snowplow-micro/

Reply:
There are a couple of links to the API on this page actually!

Reply:
If we add layers, maybe we can add a navigation item in the left nav to jump over to the API?